src/route.ts

```diff
@@ -37,10 +37,12 @@ export class Route {
    * @since 0.4.0
    */
   static parse(s: string, decode: boolean = true): Route {
-    const { pathname, searchParams } = new URL(s, 'http://localhost') // `base` is needed when `path` is relative
+    const { pathname, searchParams } = s.startsWith('/')
```

Comment:
@thewilkybarkid uhm, according to your description this change was added just for `//`: wouldn't have been better to handle that specific, edge case only?